en/docs/install-and-setup/setup/deployment-best-practices/changing-the-default-ports-with-offset.md

```diff
@@ -145,3 +148,49 @@ listenerConfigurations:
   port: http_port
 ```
 
+## Changing the default SI ports
+
+The default port offset in each of the WSO2 Streaming Integrator components is `0`. The components include the Streaming Integrator runtime, Tooling, and the Dashboard.
+
+#### Changing the Binary and Thrift ports
+
+You can offset the binary and thrift ports by configuring the offset parameter in the `<SI_HOME>|<SI_TOOLING_HOME>|<SI_DASHBOARD_HOME>/conf/server/deployment.yaml` file. 
+
+The following is a sample configuration:
+
+```yaml
+# Carbon Configuration Parameters
+wso2.carbon:
+    # value to uniquely identify a server
+id: wso2-si
+    # server name
+name: WSO2 Streaming Integrator
+    # server type
+type: wso2-si
+    # ports used by this server
+ports:
+    # port offset
+    offset: 1
```

Comment:
For SI tooling we have default port offset 3 and for SI we have it as 0

Reply:
updated